services/agent/src/simcore_service_agent/core/api/rpc/_volumes.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+from fastapi import FastAPI
+from models_library.projects_nodes_io import NodeID
+from servicelib.logging_utils import log_context
+from servicelib.rabbitmq import RPCRouter
+from simcore_service_agent.services.volumes_manager import get_volumes_manager
+
+_logger = logging.getLogger(__name__)
+
+router = RPCRouter()
+
+
+@router.expose()
+async def remove_volumes_without_backup_for_service(
+    app: FastAPI, *, node_id: NodeID
+) -> None:
+    with log_context(_logger, logging.INFO, f"removing volumes for service: {node_id}"):
+        await get_volumes_manager(app).remove_service_volumes(node_id)
+
+
+@router.expose()
```

Comment:
YOu do not expose any exceptions to the interface (see `reraise_if_error_type`)? how do you handle errors?

Reply:
Perhaps you want to expose NoServiceVolumesFoundError?

Reply:
Thanks for reminding me about it. Correct I have to expose it and use it when calling this interface.